packages/editor/src/components/post-author/post-author-select.js

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Note: this file is almost unchanged from the packages/editor/src/components/post-author/index.js as it existed prior to introducing the Combobox version, with the once exception that it now uses the `SelectControl` component.